internal/cli/usercmd/push.go

```diff
@@ -202,10 +200,17 @@ func (c *EpinioClient) Push(ctx context.Context, params PushParams) error { // n
 		log.V(3).Info("stage response", "response", stageResponse)
 
 		details.Info("start tailing logs", "StageID", stageResponse.Stage.ID)
-		err = c.stageLogs(details, appRef, stageResponse.Stage.ID)
+		c.stageLogs(appRef, stageResponse.Stage.ID)
 		if err != nil {
 			return err
 		}
+
+		details.Info("wait for job", "StageID", stageID)
+		// blocking function that wait until the staging is done
+		_, err := c.API.StagingComplete(appRef.Namespace, stageID)
+		if err != nil {
+			return errors.Wrap(err, "waiting for staging failed")
+		}
```

Comment:
Same as before

Reply:
here I don't want to return. There is more code to be run below.

Reply:
lol sorry, I missed that!